src/button-dropdown/interfaces.ts

```diff
@@ -53,6 +53,11 @@ export interface ButtonDropdownProps extends BaseComponentProps, ExpandToViewpor
    * Determines whether the button dropdown is disabled. Users cannot interact with the control if it's disabled.
    */
   disabled?: boolean;
+  /**
+   * Provides a reason why the button dropdown is disabled (only when `disabled` is `true`).
+   * If provided, the disabled button becomes focusable.
```

Comment:
I think this needs to get updated because we only support normal and primary variant. icon, and inline-icon are not supported anymore.
```suggestion
   * Applicable for normal and primary button dropdown variant.
```

Reply:
It has already documented in usage guidelines for button and button dropdown